extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -195,6 +189,20 @@ public void handle(AsyncResult<OAuth2Auth> event) {
         return new TenantConfigContext(auth, oidcConfig);
     }
 
+    @SuppressWarnings("deprecation")
+    private TenantConfigContext createdTenantContextFromPublicKey(OAuth2ClientOptions options, OidcTenantConfig oidcConfig) {
+        if (oidcConfig.applicationType == ApplicationType.WEB_APP) {
+            throw new ConfigurationException("'public-key' property can only be used with the 'service' applications");
+        }
+        LOG.info("'public-key' property for the local token verification is set,"
```

Comment:
This should really be at debug, in general we try not to pollute the log on startup, especially not messages that just inform the user of what they have done.

Reply:
@stuartwdouglas Sure, I'm not trying to always set `debug` but sometimes `info` gets in :-)